src/OBJ/Parser.elm

```diff
@@ -54,142 +63,164 @@ parseLine l =
 
 file : Parser s (List Line)
 file =
-    (many ignoredLines)
-        *> sepBy (many1 ignoredLines)
-            line
-        <* (many ignoredLines)
-        <* end
+    ignore (many ignoredLines) (sepBy (many1 ignoredLines) line)
```

Comment:
I really like this, looks so much more readable :+1:

Reply:
:smile: